### PR TITLE
WINUI-2758-Added compact-sizing.md

### PR DESCRIPTION
**Pull request by umanagarajan12**
WINUI-2758-Added compact-sizing.md and updated the images in samples.md

### CODE REVIEW

File: winui/Common/compact-sizing.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Compact sizing enables dense, information-rich groups of controls and can help with the following:
+* Browsing large amounts of content.
+* Maximizing visible content on a page.
+* Navigating and interacting with controls and content.
+
+## Examples of compact sizing
+
+Compact sizing is implemented through a special resource dictionary that can be specified in your application at either the page level or on a specific layout. The resource dictionary is available in the WinUI Nuget package.
+
+The following examples show how the Compact style can be applied for the window and an individual Grid control.
```
Comment (jeganraj-m) [inline winui/Common/compact-sizing.md:25]: show -> shows

Comment (umanagarajan12) [reply winui/Common/compact-sizing.md:25]: checked the grammatical errors and updated the changes

**Review by jeganraj-m — APPROVED** [review]

File: winui/Common/compact-sizing.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+description: Learn here about Right to left FlowDirection support in Syncfusion WinUI Project Reunion controls and more details. 
+platform: WinUI
+control: RightToLeft
+documentation: ug
+---
+
+# Compact Sizing in WinUI Controls
+
+There are two types of spacing support for controls in WinUI.
+1. [Standard](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-standard-sizing) - to accommodate both touch and pointer input
```
Comment (florenceliliana) [inline winui/Common/compact-sizing.md:13]: 1. [Standard](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-standard-sizing) - to accommodate both touch and pointer input.

File: winui/Common/compact-sizing.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+2. [Compact](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-compact-sizing) - designed primarily to accommodate pointer input
```
Comment (florenceliliana) [inline winui/Common/compact-sizing.md:14]: 2. [Compact](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-compact-sizing) - designed primarily to accommodate pointer input.

File: winui/Common/compact-sizing.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Compact sizing enables dense, information-rich groups of controls and can help with the following:
+* Browsing large amounts of content.
+* Maximizing visible content on a page.
+* Navigating and interacting with controls and content.
+
+## Examples of compact sizing
+
+Compact sizing is implemented through a special resource dictionary that can be specified in your application at either the page level or at a specific layout. The resource dictionary is available in the WinUI Nuget package.
```
Comment (florenceliliana) [inline winui/Common/compact-sizing.md:23]: Compact sizing is implemented through a unique resource dictionary that can be specified in your application at either the page level or a specific layout. The resource dictionary is available in the WinUI Nuget package.

File: winui/Common/compact-sizing.md (hunk 2 above)
Comment (umanagarajan12) [reply winui/Common/compact-sizing.md:13]: updated the changes

File: winui/Common/compact-sizing.md (hunk 3 above)
Comment (umanagarajan12) [reply winui/Common/compact-sizing.md:14]: updated the changes

File: winui/Common/compact-sizing.md (hunk 4 above)
Comment (umanagarajan12) [reply winui/Common/compact-sizing.md:23]: updated the changes

**Pull request merged**